src/MercadoPago/Client/Order/OrderTransactionClient.php

```diff
@@ -59,4 +60,23 @@ public function update(string $order_id, string $transaction_id, array $request,
         $result->setResponse($response);
         return $result;
     }
+
+    /**
+     * Method responsible for deleting transactions for an Order.
+     * @param string $order_id Order ID.
+     * @param string $transaction_id Transaction ID.
+     * @param \MercadoPago\Client\Common\RequestOptions request options to be sent.
+     * @return Response
+     * @throws \MercadoPago\Exceptions\MPApiException if the request fails.
+     * @throws \Exception if the request fails.
+     */
+    public function deleteTransaction(string $order_id, string $transaction_id, ?RequestOptions $request_options = null): MPResponse
```

Comment:
Sugiro trocarmos o retorno da função de delete para `bool`, assim podemos retornar `true` quando foi sucesso e `false` quando deu algum erro.
Já que a API não retorna response body, nós se baseamos somente no status code para a decisão de sucesso ou erro, então não vejo necessidade de retornar um `MPResponse`.
@eltinMeli o que acha?
```suggestion
    public function deleteTransaction(string $order_id, string $transaction_id, ?RequestOptions $request_options = null): bool
```

Reply:
O problema de retornar assim André, que nao conseguimos informar qual erro ocorreu junto do seu status

Reply:
Hmm verdade, valeu pelo toque!